xapi-tracing.opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
```

Comment:
Note package will need to be added to xs-opam under `packages/xs-extra`

Reply:
https://github.com/xapi-project/xs-opam/pull/637